fineract-provider/src/main/java/org/apache/fineract/accounting/rule/domain/AccountingRule.java

```diff
@@ -76,8 +84,6 @@ public class AccountingRule extends AbstractPersistableCustom {
     @Column(name = "allow_multiple_debits", nullable = false)
     private boolean allowMultipleDebitEntries;
 
-    protected AccountingRule() {}
-
     private AccountingRule(final Office office, final GLAccount accountToDebit, final GLAccount accountToCredit, final String name,
```

Comment:
Why can't this one be replaced? Is there anything more complex than setting values in that private constructor?

Reply:
done